### DISCUSSION TITLE
Please add polar.sh as a plattform option in funding.yaml

### DISCUSSION THREAD

**Discussion by matmair** [community/community#106700] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I would like to have [polar.sh](https://polar.sh/) as a platform option in the funding.yaml.
Polar is basically like Patreon but has an amazing integration with GitHub and a familiar design language. I have recently started using them in a [mid-size project](https://github.com/inventree/InvenTree/) (3k stars) and it feels much more native than Patreon or Ko-fi. Plus their source is open on [GitHub](https://github.com/polarsource/polar), which I appreciate a lot as it demonstrates that they also stand behind their engineering.

See this help topic regarding funding platforms/where I would like to see it added as an option: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository

Not affiliated with them in any way, just appreciate how well-executed they work in this vital but slow space.

**Comment by octokatherine** [comment, marked as answer]
Hi @matmair! We added the ability to have polar.sh as a funding platform. Thanks for the suggestion!

**Reply by matmair** [reply]
Thank you for the fast response!

**Reply by frankie567** [reply]
Thank you for the quick response and fast addition, @octokatherine 🙏 

~I noticed the corresponding JSON schema (https://json.schemastore.org/github-funding.json) is not yet updated. Do you know if it'll be updated there as well? 🙂~

Never mind, my colleague told me it's actually a [community-supported effort](https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/github-funding.json) 😅 I'll fix it there!

**Reply by birkjernstrom** [reply]
This is incredible news! Thank you so much @octokatherine and friends at GitHub and @matmair & our community for making this happen ♥️
